modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro

```diff
@@ -216,6 +216,32 @@ definition {
 		}
 	}
 
+	macro addTaxConfigurationFixedTaxRate {
+		CommerceEntry.gotoMenuTab(menuTab = "Tax Rates");
+
+		Click(locator1 = "Button#PLUS");
+
+		SelectFrameTop();
+
+		SelectFrame(locator1 = "IFrame#MODAL_BODY");
+
+		Select(
+			key_fieldLabel = "Tax Category",
+			locator1 = "Select#GENERIC_SELECT_FIELD",
+			value1 = ${taxCategory});
+
+		Type(
+			key_fieldLabel = ${fieldLabel},
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = ${quantity});
+
+		Button.clickSubmitButton();
+
+		Refresh();
+
+		Button.clickSave();
```

Comment:
Please avoid using the refresh function. Also, I don't think the Save is needed here